app/controllers/api/v1/gardens/env_measurements_controller.rb

```diff
@@ -0,0 +1,14 @@
+class Api::V1::Gardens::EnvMeasurementsController < ApplicationController
+  before_action :set_garden, only: [:index]
+
+	def index
+    render json: EnvMeasurementSerializer.new(@garden.env_measurements)	
```

Comment:
Changing this to EnvMeasurementSerialize definitely makes this clearer than sending it to a GardenSerializer, awesome work Rob.

Thanks for talking through this change with me. I definitely see you were right about nesting/namespacing to env_measurements how you did.